client/blocks/inline-help/contextual-help.js

```diff
@@ -766,329 +868,371 @@ const videosForSection = {
-			title: 'Embed a Tweet from Twitter in Your Website',
-			description:
+			title: translate( 'Embed a Tweet from Twitter in Your Website' ),
+			description: translate(
 				'Find out how to embed a Tweet in your content (including posts and pages) on your WordPress.com ' +
-				'or Jetpack-enabled WordPress website or blog.',
+					'or Jetpack-enabled WordPress website or blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/uVRji6bKJUE',
-			title: 'Embed an Instagram Photo in Your Website',
-			description:
+			title: translate( 'Embed an Instagram Photo in Your Website' ),
+			description: translate(
 				'Find out how to embed an Instagram photo in your content (including posts and pages) on your WordPress.com ' +
-				'or Jetpack-enabled WordPress website or blog.',
+					'or Jetpack-enabled WordPress website or blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/sKm3Q83JxM0',
-			title: 'Embed a Facebook Update in Your Website',
-			description:
+			title: translate( 'Embed a Facebook Update in Your Website' ),
+			description: translate(
 				'Find out how to embed a Facebook update in your content (including posts, pages, and even comments) on your ' +
-				'WordPress.com or Jetpack-enabled WordPress website or blog.',
+					'WordPress.com or Jetpack-enabled WordPress website or blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/SBgNkre_b14',
-			title: 'Share Blog Posts Directly on Twitter',
-			description:
-				'Find out how to share blog posts directly on Twitter from your WordPress.com or Jetpack-enabled WordPress site.',
+			title: translate( 'Share Blog Posts Directly on Twitter' ),
+			description: translate(
+				'Find out how to share blog posts directly on Twitter from your WordPress.com or Jetpack-enabled WordPress site.'
+			),
 		},
 	],
 	settings: [
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/0YCZ22k4SfQ',
-			title: 'Add a Site Logo',
-			description: 'Find out how to add a custom logo to your WordPress.com site.',
+			title: translate( 'Add a Site Logo' ),
+			description: translate( 'Find out how to add a custom logo to your WordPress.com site.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/vucZ1uZ2NPo',
-			title: 'Update Your Website Title and Tagline',
-			description:
+			title: translate( 'Update Your Website Title and Tagline' ),
+			description: translate(
 				'Find out how to update the Title and Tagline of your WordPress.com site, which you can also ' +
-				'do on your Jetpack-enabled WordPress site.',
+					'do on your Jetpack-enabled WordPress site.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/Y6iPsPwYD7g',
-			title: 'Change Your Privacy Settings',
-			description: 'Find out how to change your website privacy settings on WordPress.com.',
+			title: translate( 'Change Your Privacy Settings' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Change privacy settings' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).